lighthouse-core/lib/asset-saver.js

```diff
@@ -18,6 +18,7 @@
 
 const fs = require('fs');
 const log = require('../../lighthouse-core/lib/log.js');
+// The networkRecords artifacts have circular references
```

Comment:
I would add this comment next to the single usage so we know why that one is different.